oryx-app-mllib/src/main/java/com/cloudera/oryx/app/mllib/kmeans/KMeansEvaluation.java

```diff
@@ -166,11 +165,20 @@
       overallSilhouetteCoefficientForClustering += silhouetteCoefficientForCluster;
     }
 
-    double silhouetteCoefficient = overallSilhouetteCoefficientForClustering / data.count();
+    double silhouetteCoefficient = overallSilhouetteCoefficientForClustering / dataSize;
```

Comment:
It is the size of sample, correct? Its being passed in as method arg from parent !